docs/pages/versions/unversioned/sdk/netinfo.mdx

```diff
@@ -65,7 +65,6 @@ In order to access the `ssid` property (available under `state.details.ssid`), t
 ```
 
 - Check the **Access Wi-Fi Information** box in your app's App Identifier, [which can be found here](https://developer.apple.com/account/resources/identifiers/list)
+- Rebuild your app with `eas build --platform ios` or [`npx expo run:ios`](/workflow/expo-cli/#compiling)
```

Comment:
Do we have a similar relevant link for eas build? https://docs.expo.dev/build/introduction/?

Reply:
Yeah, we do. I think we can link to this section: https://docs.expo.dev/build/setup/#4-run-a-build since it covers both simulator/emulator, devices and app stores.